### PR TITLE
Add parameter for additional javascript files for vcf-report

### PR DESCRIPTION
**Pull request by fxwiegand**
Inspired by https://github.com/rust-bio/rust-bio-tools/pull/81, this PR introduces a new parameter `custom-js-files` that allows the user to include additional javascript files (e.g. libraries) for usage in the custom javascript template. The order of the arguments given to the vcf-report is retained for the imports in the HTML files. Beware that some javascript libraries like jQuery or Vega-Lite are already imported and that importing them again may lead to problems.

### CODE REVIEW

**Review by johanneskoester — APPROVED** [review]

File: src/bcf/report/mod.rs [hunk 1]
```diff
@@ -10,6 +12,7 @@ pub mod table_report;
 pub fn embed_js(
     output_path: &str,
     custom_table_report_js: Option<&str>,
+    custom_js_files: Option<Values>,
```
Comment (johanneskoester) [inline src/bcf/report/mod.rs:15]: Here, a Vec is sufficient. No need to wrap the values in an option. If the vec is empty, the loop below will not be entered.

File: src/bcf/report/table_report/create_report_table.rs [hunk 2]
```diff
@@ -47,6 +47,7 @@ pub(crate) fn make_table_report(
     sample: String,
     output_path: &str,
     max_read_depth: u32,
+    js_files: Option<Vec<String>>,
```
Comment (johanneskoester) [inline src/bcf/report/table_report/create_report_table.rs:50]: Same as above.

File: src/bcf/report/table_report/mod.rs [hunk 3]
```diff
@@ -20,6 +20,7 @@ pub fn table_report(
     info: Option<Values>,
     format: Option<Values>,
     max_read_depth: u32,
+    js_files: Option<Vec<String>>,
```
Comment (johanneskoester) [inline src/bcf/report/table_report/mod.rs:23]: Same as above.

**Review by johanneskoester — CHANGES_REQUESTED** [review]

File: src/bcf/report/mod.rs (hunk 1 above)
Comment (fxwiegand) [reply src/bcf/report/mod.rs:15]: Oh that is pretty smart, I didn't think of that thanks!

**Review by johanneskoester — APPROVED** [review]

**Pull request merged**